analyse_acuracy.py

```diff
@@ -0,0 +1,165 @@
+from ast import literal_eval
+import numpy as np
+import os
+import pandas as pd
+from pathlib import Path
+
+
+def parse_table(path):
+    pass
+
+
+
+
+def add_count_data(df, col, frames_col='frames'): 
+    # because frames is what the col t-min - t-max col is called
+    if isinstance(df.loc[0, col], str):
+        # this and the following isinstance lines are a tad fragile
+        # will address this at a later date
+        lens_p = [len(list(set(literal_eval(l)))) \
+                  for l in df[col].values]
+    else:
+        m = 'Incorrect type for counting false positives'
+        assert isinstance(df.loc[0, col], list), m
+        lens_p = [len(list(set(l))) \
+                  for l in df[col].values]
+    df[col + '_count'] = lens_p
+    df[col + '_per_frame'] = df[col + '_count'] / df[frames_col]
+    return df
+
+
+def generate_summary_df(dfs):
```

Comment:
Can you add a docstring here so I can figure out what you're doing? LOL I know for sure there's gonna be a better way than a for loop over the rows!